simulator-core/src/main/java/com/evbox/everon/ocpp/simulator/station/evse/ChargingStopReason.java

```diff
@@ -0,0 +1,16 @@
+package com.evbox.everon.ocpp.simulator.station.evse;
+
+/**
+ * Reason why charging in EVSE stopped.
+ *
+ * LOCALLY_STOPPED corresponds to charge locally stopped by user/token
+ * REMOTELY_STOPPED corresponds to charge stopped remotely
+ */
+public enum ChargingStopReason {
+    LOCALLY_STOPPED,
```

Comment:
Can we add mappings of each reason with corresponding TransactionData.StoppedReason and remove conditional logic from Unplug (I mean `evse.getStopReason() != null && evse.getStopReason().isRemotelyStopped()`) ?